.editorconfig

```diff
@@ -17,3 +17,6 @@ max_line_length = 80
 
 [COMMIT_EDITMSG]
 max_line_length = 0
+
+[*.txt]
+trim_trailing_whitespace = false
```

Comment:
please remove, we don't want that rule on every txt file
we can live with a few warnings in fixtures

Reply:
Done